Configuration/PyReleaseValidation/python/relval_2026.py

```diff
@@ -39,7 +39,7 @@
 numWFIB.extend([38234.0]) #2026D85
 numWFIB.extend([38634.0]) #2026D86
 numWFIB.extend([39034.0]) #2026D87
-numWFIB.extend([39434.0]) #2026D88
+numWFIB.extend([39434.0,39434.5,39434.501,39434.502]) #2026D88, pixelTrackingOnly, Patatrack local reconstruction on CPU, Patatrack local reconstruction on GPU
```

Comment:
In analogy to what is done in the other lines of this config file:
```suggestion
numWFIB.extend([39434.0) #2026D88
numWFIB.extend([39434.5,39434.501,39434.502]) #2026D88 pixelTrackingOnly, Patatrack local reconstruction on CPU, Patatrack local reconstruction on GPU
```